flask_jwt_router/_routing.py

```diff
@@ -116,26 +117,40 @@ def _allow_public_routes(self, white_routes):
                 return False
         return True
 
+    def _is_route_exist(self, url: str, method: str) -> bool:
```

Comment:
can we name this method: `_does_route_exist` please.

Reply:
Renamed